{"code":"SQLITE_CANTOPEN","errno":14,"level":"error","message":"Erreur de connexion à la base de données: SQLITE_CANTOPEN: unable to open database file","stack":"Error: SQLITE_CANTOPEN: unable to open database file","timestamp":"2025-06-23T04:44:29.961Z"}
{"code":"SQLITE_CANTOPEN","errno":14,"level":"error","message":"Erreur de connexion à la base de données: SQLITE_CANTOPEN: unable to open database file","stack":"Error: SQLITE_CANTOPEN: unable to open database file","timestamp":"2025-06-23T04:53:23.024Z"}
{"code":"SQLITE_CANTOPEN","errno":14,"level":"error","message":"Erreur de connexion à la base de données: SQLITE_CANTOPEN: unable to open database file","stack":"Error: SQLITE_CANTOPEN: unable to open database file","timestamp":"2025-06-23T05:06:39.066Z"}
{"code":"SQLITE_CANTOPEN","errno":14,"level":"error","message":"Erreur de connexion à la base de données: SQLITE_CANTOPEN: unable to open database file","stack":"Error: SQLITE_CANTOPEN: unable to open database file","timestamp":"2025-06-23T05:11:58.958Z"}
{"code":"SQLITE_CANTOPEN","errno":14,"level":"error","message":"Erreur de connexion à la base de données: SQLITE_CANTOPEN: unable to open database file","stack":"Error: SQLITE_CANTOPEN: unable to open database file","timestamp":"2025-06-23T05:13:55.810Z"}
{"code":"SQLITE_CANTOPEN","errno":14,"level":"error","message":"Erreur de connexion à la base de données: SQLITE_CANTOPEN: unable to open database file","stack":"Error: SQLITE_CANTOPEN: unable to open database file","timestamp":"2025-06-23T05:25:08.610Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur lors de la recherche de l’utilisateur: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T15:57:05.188Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur d’authentification: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T15:57:05.190Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur lors de la recherche de l’utilisateur: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T15:57:18.607Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur d’authentification: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T15:57:18.609Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur lors de la recherche de l’utilisateur: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T15:57:23.560Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur d’authentification: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T15:57:23.563Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur lors de la recherche de l’utilisateur: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T15:57:28.137Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur d’authentification: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T15:57:28.140Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur lors de la recherche de l’utilisateur: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T16:03:42.786Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur d’authentification: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_app\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_app\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T16:03:42.788Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur lors de la recherche de l’utilisateur: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T20:08:39.883Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur d’authentification: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T20:08:39.884Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur lors de la recherche de l’utilisateur: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T20:14:30.913Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur d’authentification: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T20:14:30.914Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur lors de la recherche de l’utilisateur: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T20:16:57.941Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur d’authentification: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T20:16:57.943Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur lors de la recherche de l’utilisateur: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T20:19:04.486Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur d’authentification: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T20:19:04.487Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur lors de la recherche de l’utilisateur: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin ' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T20:21:29.835Z"}
{"__augmented":true,"code":"SQLITE_ERROR","errno":1,"level":"error","message":"Erreur d’authentification: SQLITE_ERROR: no such table: users","stack":"Error: SQLITE_ERROR: no such table: users\n--> in Database#get('SELECT * FROM users WHERE username = ?', [ 'admin ' ], [Function (anonymous)])\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:48:10\n    at new Promise (<anonymous>)\n    at User.findByUsername (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\database\\models\\User.js:47:12)\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\services\\AuthService.js:8:29)\n    at AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\controllers\\AuthController.js:7:38)\n    at C:\\Users\\Jarvis\\Desktop\\berto_tracker\\main.js:42:31\n    at Session.<anonymous> (node:electron/js2c/browser_init:2:106839)\n    at Session.emit (node:events:518:28)","timestamp":"2025-06-23T20:21:29.839Z"}
{"level":"error","message":"Erreur d’authentification: Mot de passe incorrect","stack":"Error: Mot de passe incorrect\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\services\\AuthService.js:15:13)\n    at async AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\controllers\\AuthController.js:7:20)\n    at async C:\\Users\\Jarvis\\Desktop\\berto_tracker\\main.js:42:10\n    at async Session.<anonymous> (node:electron/js2c/browser_init:2:106823)","timestamp":"2025-06-23T20:51:41.557Z"}
{"level":"error","message":"Erreur d’authentification: Mot de passe incorrect","stack":"Error: Mot de passe incorrect\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\services\\AuthService.js:15:13)\n    at async AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\controllers\\AuthController.js:7:20)\n    at async C:\\Users\\Jarvis\\Desktop\\berto_tracker\\main.js:42:10\n    at async Session.<anonymous> (node:electron/js2c/browser_init:2:106823)","timestamp":"2025-06-23T20:51:47.061Z"}
{"level":"error","message":"Erreur d’authentification: Mot de passe incorrect","stack":"Error: Mot de passe incorrect\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\services\\AuthService.js:15:13)\n    at async AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\controllers\\AuthController.js:7:20)\n    at async C:\\Users\\Jarvis\\Desktop\\berto_tracker\\main.js:42:10\n    at async Session.<anonymous> (node:electron/js2c/browser_init:2:106823)","timestamp":"2025-06-23T20:52:01.621Z"}
{"level":"error","message":"Erreur d’authentification: Utilisateur non trouvé","stack":"Error: Utilisateur non trouvé\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\services\\AuthService.js:10:13)\n    at async AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\controllers\\AuthController.js:7:20)\n    at async C:\\Users\\Jarvis\\Desktop\\berto_tracker\\main.js:42:10\n    at async Session.<anonymous> (node:electron/js2c/browser_init:2:106823)","timestamp":"2025-06-24T06:37:46.034Z"}
{"level":"error","message":"Erreur d’authentification: Utilisateur non trouvé","stack":"Error: Utilisateur non trouvé\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\services\\AuthService.js:10:13)\n    at async AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\controllers\\AuthController.js:7:20)\n    at async C:\\Users\\Jarvis\\Desktop\\berto_tracker\\main.js:42:10\n    at async Session.<anonymous> (node:electron/js2c/browser_init:2:106823)","timestamp":"2025-06-24T15:38:54.213Z"}
{"level":"error","message":"Erreur d’authentification: Utilisateur non trouvé","stack":"Error: Utilisateur non trouvé\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\services\\AuthService.js:10:13)\n    at async AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\controllers\\AuthController.js:7:20)\n    at async C:\\Users\\Jarvis\\Desktop\\berto_tracker\\main.js:42:10\n    at async Session.<anonymous> (node:electron/js2c/browser_init:2:106823)","timestamp":"2025-06-24T15:39:00.424Z"}
{"level":"error","message":"Erreur d’authentification: Utilisateur non trouvé","stack":"Error: Utilisateur non trouvé\n    at AuthService.authenticate (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\services\\AuthService.js:10:13)\n    at async AuthController.login (C:\\Users\\Jarvis\\Desktop\\berto_tracker\\src\\backend\\controllers\\AuthController.js:7:20)\n    at async C:\\Users\\Jarvis\\Desktop\\berto_tracker\\main.js:42:10\n    at async Session.<anonymous> (node:electron/js2c/browser_init:2:106823)","timestamp":"2025-06-24T15:39:12.620Z"}
